crates/debug/src/transform/attr.rs

```diff
@@ -243,17 +240,14 @@ where
             AttributeValue::CallingConvention(e) => write::AttributeValue::CallingConvention(e),
             AttributeValue::Inline(e) => write::AttributeValue::Inline(e),
             AttributeValue::Ordering(e) => write::AttributeValue::Ordering(e),
-            AttributeValue::UnitRef(ref offset) => {
-                if let Some(unit_id) = die_ref_map.get(offset) {
-                    write::AttributeValue::ThisUnitEntryRef(*unit_id)
-                } else {
-                    pending_die_refs.push((current_scope_id, attr.name(), *offset));
-                    continue;
-                }
+            AttributeValue::UnitRef(offset) => {
+                pending_die_refs.insert(current_scope_id, attr.name(), offset);
+                continue;
+            }
+            AttributeValue::DebugInfoRef(offset) => {
+                pending_di_refs.insert(current_scope_id, attr.name(), offset);
+                continue;
             }
-            // AttributeValue::DebugInfoRef(_) => {
-            //     continue;
-            // }
             _ => panic!(), //write::AttributeValue::StringRef(out_strings.add("_")),
```

Comment:
Unrelated to this change, but we should probably return an error here rather than panicking.

Reply:
Filed #914